src/Cake.Core/IO/FileExtensions.cs

```diff
@@ -100,5 +100,49 @@ public static IEnumerable<string> ReadLines(this IFile file, System.Text.Encodin
                 return result;
             }
         }
+
+        /// <summary>
+        /// Checks if file has CLR PE Header.
+        /// </summary>
+        /// <param name="file">The file to be read from.</param>
+        /// <returns><c>true</c> if file is CLR assembly; otherwise, <c>false</c>.</returns>
+        public static bool IsCLRAssembly(this IFile file)
```

Comment:
Can we rename this to IsClrAssembly now when it's public?

Reply:
Sure